packages/runtime/container-runtime/src/summarizerTypes.ts

```diff
@@ -309,7 +311,7 @@ export interface ISummarizer extends
     /* Closes summarizer. Any pending processes (summary in flight) are abandoned. */
     close(): void;
 
-    run(onBehalfOf: string, options?: Readonly<Partial<ISummarizerOptions>>): Promise<SummarizerStopReason>;
+    run(onBehalfOf: string, disableHeuristics?: boolean): Promise<SummarizerStopReason>;
```

Comment:
Why do we need to overwrite that setting if class implementing this interface already has full config? See my other comment around newConfig - I believe this should not be required any more (unless we make this decision on the spot for some reason)

Reply:
I left a question on the other comment :)